src/modules/rc_update/rc_update.cpp

```diff
@@ -504,6 +506,7 @@ switch_pos_t RCUpdate::get_rc_sw2pos_position(uint8_t func, float on_th) const
 {
 	if (_rc.function[func] >= 0) {
 		const bool on_inv = (on_th < 0.f);
+		on_th = fabsf(on_th);
```

Comment:
Below both `value < on_th` and `value > on_th` is checked.

Reply:
Yes, but I'm not sure what you mean?

If `on_th` is negative, `on_inv` will be true. Then below:

    if (on_inv ? value < on_th : value > on_th) {

the "then" clause of the inline-if statement will be evaluated:

     value < on_th

This will always be false, since `value` is positive and `on_th` is negative.

Perhaps you mean you can omit the `fabs()` call and change the `if` statement to:

    if (on_inv ? value < -on_th : value > on_th) {

(note the minus sign in front of `on_th`). I find it more clear with `fabs()`, which is also the way it originally was, so I left it like that. Feel free to improve!

Reply:
Yes I was suggesting omitting the abs and updating the value checks instead.